purchase_requisition_line_description/models/product.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.model
+    def _name_search(
```

Comment:
IMO this looks like a specific behavior we don't necessarily want in an OCA module.